bom/pom.xml

```diff
@@ -29,70 +47,64 @@
         <classifier>no_aop</classifier>
       </dependency>
       <dependency>
-        <groupId>com.google.inject</groupId>
-        <artifactId>guice</artifactId>
-        <version>${project.version}</version>
-        <classifier>classes</classifier>
```

Comment:
This entry should be kept in the BOM as it represents the original non-jarjar'd classes for people who want to consume Guice and use it with a different version of CGLIB/ASM.